packages/tools/src/types/AnnotationTypes.ts

```diff
@@ -81,6 +81,8 @@ type Annotation = {
     [key: string]: unknown;
     /** Cached Annotation statistics which is specific to the tool */
     cachedStats?: Record<string, unknown>;
+    /** Label of an annotation */
+    label?: string;
```

Comment:
This is inconsistent with the label in LengthTool - label belongs to the data element.   Note the difference between CS3D and OHIF - in CS3D the data is in the data object, while in OHIF it is at the top level.

Reply:
Sorry, was seeing the nesting wrong.  Could you move the handles to it's own type - I think that is sufficiently long to be worth having by itself.  That would resolve the issue with having it hard to see the nesting.

Reply:
Thanks, I like that a lot better.  I never like having to try to manually figure out the indentation to figure out what is what.